addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -989,6 +989,18 @@ registry.anchorSlide = publicWidget.Widget.extend({
         if (!utils.isValidAnchor(hash)) {
             return;
         }
+        if (hash === '#top' || hash === '#bottom') {
+            // If the anchor targets #top or #bottom, directly call the
+            // "scrollTo" function. The reason is that the header or the footer
+            // could have been removed from the DOM. By receiving a string as
+            // parameter, the "scrollTo" function handles the scroll to the top
+            // or to the bottom of the document even if the header or the
+            // footer is removed from the DOM.
+            dom.scrollTo(hash, {
+                duration: 500,
+                extraOffset: this._computeExtraOffset(),
+            });
+        }
```

Comment:
- Should not you `return` after the call to `scrollTo` to avoid executing the rest of this function?
- Actually can't you put this before the `isValidAnchor` as well?